src/index.html

```diff
@@ -1,23 +1,102 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<meta name="description" content="Update your description here" />
-	<title>Update your title here</title>
+	<title>Find your cocktail</title>
 	<!--
 		Webpack will automatically link your CSS file for you.
 		Work in the src/css/styles.scss file.
 	-->
 </head>
+
 <body>
 	<main class="content-container">
-		<h1 class="page-heading">Welcome to your final project!</h1>
+		<div class="fixed">
+			<p class="instr">Please choose your...</p>
+			<img src="https://stat.ameba.jp/user_images/20211111/08/halfkneazle/3d/9c/p/o0835062615029669704.png?caw=800"
```

Comment:
Don't forget to move this and other images into your images folder after https://github.com/ProfessorKolodziej/final-project-mix-match/pull/5 is merged and you update your branch :)